llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -937,6 +937,16 @@ def TuneLUIADDIFusion
     : SubtargetFeature<"lui-addi-fusion", "HasLUIADDIFusion",
                        "true", "Enable LUI+ADDI macrofusion">;
 
+def TuneAUIPCADDIFusion
+    : SubtargetFeature<"auipc-addi-fusion", "HasAUIPCADDIFusion",
+                       "true", "Enable AUIPC+ADDI macrofusion">;
+def TuneSLLISRLIFusion
```

Comment:
This is pretty generic name given the specific cases that are recognized for fusion.

Reply:
how about `ShiftedZExtFusion`?

Reply:
ok